<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDialogIndexaciyaType</class>
 <widget class="QDialog" name="UDialogIndexaciyaType">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Тип індексації працівників</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <item>
      <widget class="QToolButton" name="toolButton_prevYear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&lt;&lt;</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1leftarrow-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1leftarrow-256.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Рік</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBox_curYear">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="minimum">
        <number>2003</number>
       </property>
       <property name="maximum">
        <number>2100</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_nextYear">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>&gt;&gt;</string>
       </property>
       <property name="icon">
        <iconset resource="../uPictRc.qrc">
         <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1rightarrow-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1rightarrow-256.png</iconset>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_indexType">
     <property name="selectionMode">
      <enum>QAbstractItemView::SingleSelection</enum>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <column>
      <property name="text">
       <string>npr_id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>01</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>02</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>03</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>04</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>05</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>06</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>07</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>08</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>09</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>10</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>11</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>12</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
  <action name="action_basingScriptDefinition">
   <property name="icon">
    <iconset resource="../uPictRc.qrc">
     <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1day-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/1day-256.png</iconset>
   </property>
   <property name="text">
    <string>Визначається скриптом</string>
   </property>
   <property name="toolTip">
    <string>Встановити для поточного працівника базовість, що визначається скриптом</string>
   </property>
  </action>
  <action name="action_baseMonth">
   <property name="text">
    <string>Базовий місяць</string>
   </property>
   <property name="toolTip">
    <string>Встановити для поточного працівника місяць базовим</string>
   </property>
  </action>
  <action name="action_noBaseMonth">
   <property name="text">
    <string>Не базовий місяць</string>
   </property>
   <property name="toolTip">
    <string>Встановити для поточного працівника місяць не базовим</string>
   </property>
  </action>
  <action name="action_basingScriptDefinitionAll">
   <property name="icon">
    <iconset resource="../uPictRc.qrc">
     <normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/7days-256.png</normaloff>:/crystalProject/pictures/crystal-project-actions-PNG/7days-256.png</iconset>
   </property>
   <property name="text">
    <string>Базовість місяця визначається скриптом для усіх працівників</string>
   </property>
   <property name="toolTip">
    <string>Встановити для усіх працівників базовість, що визначається скриптом</string>
   </property>
  </action>
  <action name="action_baseMonthAll">
   <property name="text">
    <string>Базовий місяць для усіх працівників</string>
   </property>
   <property name="toolTip">
    <string>Встановити для усіх працівників місяць базовим</string>
   </property>
  </action>
  <action name="action_noBaseMonthAll">
   <property name="text">
    <string>Не базовий місяць для усіх працівників</string>
   </property>
   <property name="toolTip">
    <string>Встановити для усіх працівників місяць не базовим</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../uPictRc.qrc"/>
 </resources>
 <connections/>
</ui>
